src/main/java/uk/nhs/digital/nhsconnect/lab/results/utils/JmsHeaders.java

```diff
@@ -1,5 +1,9 @@
 package uk.nhs.digital.nhsconnect.lab.results.utils;
 
+import lombok.Getter;
+
 public final class JmsHeaders {
-    public static String CONVERSATION_ID = "ConversationId";
+    @Getter
+    private static String conversationId = "ConversationId";
```

Comment:
I think the fix here should have just been to make it `final`
```suggestion
    public static final String CONVERSATION_ID = "ConversationId";
```

Reply:
Done in c78621f85